clarity/contracts/arkadiko-freddie-v1-1.clar

```diff
@@ -623,6 +623,12 @@
 
     (try! (pay-stability-fee vault-id coll-type))
     (burn-partial-debt vault-id (min-of debt (get debt vault)) reserve ft coll-type)
+    ;; (let ((fee (unwrap-panic (pay-stability-fee vault-id coll-type))))
```

Comment:
Can you explain why this was removed?

Reply:
Yes sorry, this was actually added by me, but it wasn't useful in the end I think.. gonna remove this completely & merge